snippets/as/simple-storage/run.sh

```diff
@@ -0,0 +1,2 @@
+yarn --no-lockfile
+yarn build
```

Comment:
For CI purposes, we should make sure that the tests run as well

Reply:
They now run